README.md

```diff
@@ -5,7 +5,7 @@
   Apple Silicon Guide
 </h1>
 
-#### A guide covering Apple Silicon including the applications, libraries and tools that will make you a better and more efficient with your Apple Silicon powered device.
+#### A guide covering Apple Silicon including the applications, libraries and tools that will make you a better and more efficient power user with your Apple Silicon powered device.
```

Comment:
#### A guide covering Apple Silicon including the applications, libraries and tools that will make you better and more efficient with your Apple Silicon powered device.